napari/main.py

```diff
@@ -0,0 +1,40 @@
+"""
+napari command line viewer.
+"""
+import argparse
+import numpy as np
+from skimage import io
+
+from .util import app_context
+from . import Viewer
+
+
+def main():
+    parser = argparse.ArgumentParser(usage=__doc__)
+    parser.add_argument('images', nargs='+', help='Images to view.')
+    parser.add_argument(
+        '--layers',
+        action='store_true',
+        help='Treat multiple input images as layers.',
+    )
+    parser.add_argument(
+        '-m',
+        '--multichannel',
+        help='Treat images as RGB.',
+        action='store_true',
+    )
+    args = parser.parse_args()
+    with app_context():
+        v = Viewer(hide_menubar=False)
+        images = io.ImageCollection(args.images, conserve_memory=False)
```

Comment:
if someone passes no images can you launch the empty viewer? Fine if you want to wait on this till future PRs (say when we add drag / drop capability), but also maybe nice just to get it in now